.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
isn't this a personal config file? maybe add .vscode to .gitignore instead?

Reply:
Yes, removed.